uaclient/cli/__init__.py

```diff
@@ -55,8 +57,34 @@
     system_command,
 ]
 
+HelpEntry = NamedTuple(
+    "HelpEntry", [("position", int), ("name", str), ("help_string", str)]
+)
+
 
 class ProArgumentParser(argparse.ArgumentParser):
+    help_entries = OrderedDict(
+        [
+            (messages.CLI_HELP_HEADER_QUICK_START, []),
```

Comment:
Even though I get that idea, but I think we should use a `Enum` here instead. Each Enum value could have a message associated with it, but I think it creates a better separation between the helper categories and the messages module

Reply:
I think I agree - an enum would make it clearer what are allowed values when creating editing or adding sub commands.

Reply:
will do